listen_go111.go

```diff
@@ -1,4 +1,4 @@
-// +build go1.11
+// +build go1.11 go1.12
```

Comment:
`go1.xx` already means go1.xx+. This (and the `listen_go_not111.go` change) are unneeded.